solver/src/liquidity/offchain_orderbook.rs

```diff
@@ -28,6 +33,23 @@ struct OrderSettlementHandler {
     order: Order,
 }
 
+fn inflight_order_filter(order: Order, inflight_trades: &HashSet<OrderUid>) -> Option<Order> {
+    // TODO - could model inflight_trades as HashMap<OrderUid, Trade>
```

Comment:
I think this might be needed:
```suggestion
    // TODO - could model inflight_trades as HashMap<OrderUid, Vec<Trade>>
```

See comment (https://github.com/gnosis/gp-v2-services/pull/662#discussion_r642393327). This is just a comment, so not really important. Just wanted to point out that partially fillable orders will have some weirdness and hard edge cases when we get around to it.